src/components/Checkbox/index.native.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import BaseCheckbox from './BaseCheckbox';
+
+const Checkbox = props => (
+    <BaseCheckbox
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        onPress={props.onPress ? props.onPress : props.onMouseDown}
+        onMouseDown={undefined}
```

Comment:
Wouldn't this break with a mouse connected to an android phone/tablet?

Reply:
I don't think so because the mouse click must also fire the onPress event as it does on the web version as well. On web, when the mouse is clicked, both onPress and onMouseDown as fired, so I wouldn't expect any reason for a different behaviour with a mouse connected to android..